AUTHORS

```diff
@@ -62,6 +62,7 @@ answer newbie questions, and generally made Django that much better:
     andy@jadedplanet.net
     Anssi Kääriäinen <akaariai@gmail.com>
     ant9000@netwise.it
+    Anton Samarchyan <desecho@gmail.com>
```

Comment:
Please don't add this is every one of your PRs. It'll be added (sorted correctly) as part of https://github.com/django/django/pull/7754.